server/src/PendingCeremonies.hs

```diff
@@ -70,7 +70,7 @@ data ExpiringChallenge = ExpiringChallenge
 -- | The current time in seconds for challenge expiration
 getNow :: IO Int64
 getNow =
-  -- We're only interested in second-resolution, this is also why we use the Coarse clock version
+  -- We're only interested in second-resolution
```

Comment:
We're not using the coarse version because it's not portable, see https://github.com/tweag/webauthn/pull/60

Reply:
Ahh, never mind,I thought you _added_ that line, my mistake!

Reply:
That happens ;)